build.sbt

```diff
@@ -4,34 +4,25 @@ version := "0.4.2"
 
 organization := "com.databricks"
 
-scalaVersion := "2.11.7"
+scalaVersion := "2.11.12"
 
 spName := "databricks/spark-xml"
 
-crossScalaVersions := Seq("2.10.5", "2.11.7")
+crossScalaVersions := Seq("2.11.12", "2.12.8")
 
-sparkVersion := "2.0.0"
```

Comment:
I'm not sure about this, but this seemed to hard-code Spark versions to 2.0.0 despite the command line setting.